lib/taste_tester/noop.rb

```diff
@@ -22,7 +22,7 @@ class NoOp
     include TasteTester::Logging
     include BetweenMeals::Util
 
-    attr_reader :output
+    attr_reader :output, :status
```

Comment:
Should we also initialize `@status = 0` so we don't run error() unnecessarily?

Reply:
Line 56 always returns 0

Reply:
Looking through the code, these members aren't used much. They're only valid if transport.run is called first so I've refactored the code to return these directly.